common/stringUtils.js

```diff
@@ -19,7 +19,7 @@ const contains = (value = '', string = '') => R.includes(toLower(value), toLower
 const isBlank = R.ifElse(
   isString,
   R.pipe(trim, R.isEmpty),
-  R.always(false)
+  R.isNil
 )
```

Comment:
what if you pass a non empty object? will it return true?

Reply:
It returns false, as before